### DISCUSSION TITLE
Meeting not showing in the app

### DISCUSSION THREAD

**Discussion by niagaraintergroup** [code4recovery/12-step-meeting-list#1304] [Q&A] [answered]
We are having a problem loading **one** meeting into the meeting guide app.  The meeting is listed on the website but not on the app.

**Comment by joshreisner** [comment, marked as answer]
hi @niagaraintergroup the thing to do is to email [appsupport@aa.org](mailto:appsupport@aa.org) - they can help figure out what's going on

**Reply by niagaraintergroup** [reply]
Thank You!

**Reply by SIA-WEBSUP** [reply]
Just out of curiosity,
     What's the meeting?
     What's the JSON feed you provide to the meeting guide app? Is it in the feed?
